cores/esp32/HardwareSerial.h

```diff
@@ -73,9 +73,21 @@ class HardwareSerial: public Stream
     HardwareSerial(int uart_nr);
     ~HardwareSerial();
 
-    // onReceive will setup a callback for whenever UART data is received
+    // onReceive will setup a callback for whenever UART data is received and a later timeout occurs (no receiving data after specified time):
+    // param function        is the callback to be called after reception timeout
+    
     // it will work as UART Rx interrupt -- Using C++ 11 std::fuction 
     void onReceive(OnReceiveCb function);
+
```

Comment:
Please review the name of this function. 
Arduino uses function names such as **onSomething()** only when setting a callback function.
Please use **setSomething()** instead.

Reply:
Will do.